ClassicPress-research/classic-commerce
# Add files for Update Manager functionality

### All Submissions:


### Changes proposed in this Pull Request:

A file was added that handles the update manager functionality for updates that are served from classiccommerce.cc. 

### How to test the changes in this Pull Request:

1. Install the new file in the includes folder and add the call to it in classic-commerce.php on a test site.
2. Visit the test site and check for updates.
3. The current version is set at 1.0.1, so the test site should show it needs an update as shown:

![updates](https://user-images.githubusercontent.com/46998578/69491707-d95c5880-0eec-11ea-833e-f0697e928729.jpg)

### Other information:




### Changelog entry

Add file for Update Manager functionality for updates served from classiccommerce.cc.
